components/Radio/Radio.tsx

```diff
@@ -49,6 +50,8 @@ const StyledRadio = styled(RadioGroupPrimitive.Item, {
   color: '$hiContrast',
   boxShadow: 'inset 0 0 0 1px $colors$radioBorder',
   overflow: 'hidden',
+}
+const StyledRadio = styled(RadioGroupPrimitive.Item, RADIO_BASE_STYLES, {
```

Comment:
Just one minor suggestion, the Radio button has a hover effect, do we want to have it on the Accordion too? 

![Screen Recording 2022-01-24 at 12 52 54](https://user-images.githubusercontent.com/38889364/150818627-7cdcb285-6240-41c9-a656-2dba0dba89fe.gif)

Reply:
@paulocfjunior I think it's a good suggestion to add the same hover effect. To me, it should be applied when hovering over the whole accordion line, to make it clear there's no different behaviour whether you click the "radio decoration" or the rest of the line.

Would it be good for you?

https://user-images.githubusercontent.com/3367393/150834669-a4d75f02-4ee3-4775-9d4d-47c643a5e158.mp4

Reply:
Crap...